surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/Validate.java

```diff
@@ -9,4 +9,12 @@ public static boolean isEmpty(String string){
     public static boolean isNotEmpty(String string){
         return !isEmpty(string);
     }
+
+    public static boolean isDisableSmartTesting() {
```

Comment:
This can be one line and if so it probably shouldn't belong to this class but can be called directly where we need that.